files/anitya_cron.py

```diff
@@ -41,28 +51,8 @@ def main(debug):
     LOG.addHandler(fhand)
 
     projects = anitya.lib.model.Project.all(session)
-
-    if PBAR:
-        widgets = ['Release update: ',
-                   Percentage(),
-                   ' ',
-                   Bar(marker=RotatingMarker()),
-                   ' ',
-                   ETA()]
-        pbar = ProgressBar(widgets=widgets, maxval=len(projects)).start()
-
-    cnt = 0
-    for project in projects:
-        LOG.info(project.name)
-        try:
-            anitya.check_release(project, session)
-        except anitya.lib.exceptions.AnityaException as err:
-            pass
-        cnt += 1
-        if PBAR:
-            pbar.update(cnt)
-    if PBAR:
-        pbar.finish()
+    p = multiprocessing.Pool(anitya.app.APP.config.get('CRON_POOL', 10))
```

Comment:
Can you check to see if there is a performance difference between ``multiprocessing.Pool`` and ``multiprocessing.pool.ThreadPool``?